advanced-analytics-hybrid/streamlit/Dashboard.py

```diff
@@ -9,7 +9,7 @@
 def main():
 
     _WAREHOUSE = 'snowflake'
```

Comment:
Can you alter the _WAREHOUSE to CHANGE_ME like the others with multiple supported please